Makefile

```diff
@@ -4,4 +4,4 @@ build:
 test:
 	pip3 install -r test_requirements.txt
 	flake8 --exclude lib
-	pytest -v --cov server --html=report.html
+	pytest -v --cov-report term-missing --cov=server --html=report.html
```

Comment:
I always add `--cov-report term-missing` locally, but I don't normally commit. Is it okay to add it in?

Reply:
Why would it not be?  Genuine question because it looks reasonable, but could have an effect that I don't know about

Reply:
It's just that it prints a load of extra output at the end, which can be annoying when you're looking through test failures

Reply:
That's fair, I'll remove it.  Would rather travis's output be clean and useful